tests/integration/test.bulk_docs.js

```diff
@@ -162,6 +162,21 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it("7829 bare rev 1- with new_edits=false", function (done) {
+      var db = new PouchDB(dbs.name);
+      var docs = [
+        {
+          _id: "foo",
+          _rev: "1-",
+          integer: 1
+        }
+      ];
+      db.bulkDocs({ docs: docs }, { new_edits: false }, function (err) {
+        should.not.exist(err, 'no error');
+        done();
```

Comment:
Actually just a note, we can use promises here, 

    return db.bulkDocs({ docs: docs }, { new_edits: false });

Will throw if there was an error and fail the tests, no need to call `done()` manually, will merge for now but if you want to clean in a follow up would be ideal, cheers